pandas/tools/plotting.py

```diff
@@ -2262,11 +2262,13 @@ def result(self):
 
 def _plot(data, x=None, y=None, subplots=False,
           ax=None, kind='line', **kwds):
+
+    invalid_ct_fmtstr = "'%s' is not a valid chart type"
```

Comment:
I would use the word `kind` instead of `type` to match the keyword argument.

Also, perhaps use `%r` instead of `'%s'` to properly format non-string arguments, too